### PR TITLE
tc: Add release note 8851

### PR DESCRIPTION
**Pull request by tara-det-ai**
## Description

Adds release note for #8851 force default user passwords for all det deploy and CI clusters

### CODE REVIEW

File: docs/release-notes/8851-force-default-passwords.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**New Features**
+
+-  Add a configuration setting, ``initial_user_password``, to the master configuration file forcing the setup of an initial user password for the built-in ``determined`` and ``admin`` users during the first launch, specifically when a cluster's database is bootstrapped.
```
Comment (stoksc) [inline docs/release-notes/8851-force-default-passwords.rst:5]:
```suggestion
-  Add a configuration setting, ``initial_user_password``, to the master configuration file forcing the setup of an initial user password for the built-in ``determined`` and ``admin`` users during the first launch, specifically when a cluster's database is bootstrapped. Users with existing clusters should ensure they have passwords set for all users if those clusters are publicly accessible.
```

File: docs/release-notes/8851-force-default-passwords.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**New Features**
```
Comment (stoksc) [inline docs/release-notes/8851-force-default-passwords.rst:3]:
```suggestion
**Security**
```

**Review by stoksc — APPROVED** [review]
lgtm w/ comments.

this needs to go into the current release with the change i landed, so i'm going to mark it "needs cherry pick". we should chat offline about some other details, though.

File: docs/release-notes/8851-force-default-passwords.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/release-notes/8851-force-default-passwords.rst]: added the changes https://github.com/determined-ai/determined/pull/8864/commits/e193924793ea6c32de91ddacf4b575cef34eefc3

File: docs/release-notes/8851-force-default-passwords.rst (hunk 2 above)
Comment (tara-det-ai) [reply docs/release-notes/8851-force-default-passwords.rst]: added https://github.com/determined-ai/determined/pull/8864/commits/e193924793ea6c32de91ddacf4b575cef34eefc3

**Pull request merged**